cf-agent/verify_files_utils.c

```diff
@@ -836,7 +837,7 @@ static PromiseResult SourceSearchAndCopy(EvalContext *ctx, const char *from, cha
             }
         }
 
-        if (attr.copy.purge)    /* Purge this file */
+        if (attr.copy.purge)    /* Do not purge this file */
```

Comment:
Really? Later on it tests for the same condition and calls `PurgeLocalFiles()`. Seems to me like `attr.copy.purge` *does* indicate that the file should be purged. It is true that it doesn't happen here though, so perhaps a more fitting comment would be `/* Indicates that file should be purged */`.